worker/migrationmaster/worker.go

```diff
@@ -432,12 +438,66 @@ func (w *Worker) doSUCCESS(status coremigration.MigrationStatus) (coremigration.
 	return coremigration.LOGTRANSFER, nil
 }
 
-func (w *Worker) doLOGTRANSFER() (coremigration.Phase, error) {
-	// TODO(mjs) - To be implemented.
-	// w.setInfoStatus("successful: transferring logs to target controller")
+func (w *Worker) doLOGTRANSFER(targetInfo coremigration.TargetInfo, modelUUID string) (coremigration.Phase, error) {
+	err := w.transferLogs(targetInfo, modelUUID)
+	if err != nil {
+		return coremigration.UNKNOWN, errors.Trace(err)
+	}
 	return coremigration.REAP, nil
 }
 
+func (w *Worker) transferLogs(targetInfo coremigration.TargetInfo, modelUUID string) error {
+	sent := 0
+	reportProgress := func(sent int) {
+		w.setInfoStatus("successful, transferring logs to target controller (%d sent)", sent)
```

Comment:
If it writes to the log file, then maybe. Can we reduce the number of times we write it out? Some models may have a lot of logs, and writing each thousand may be too often. Can we report percentages? Do we have a count at the start?